scripts/lib.sh

```diff
@@ -11,10 +11,10 @@ ensure_value() {
     # set naked value
     if grep -q "^$name=" "$file" 2>/dev/null; then
         # use '|' sed delimiter as we use '/' in values
-        sed -i "s|^$name=.*|$name=\"$value\"|" "$file"
+        gsed -i "s|^$name=.*|$name=\"$value\"|" "$file"
```

Comment:
I don't have gsed installed and the setup instructions don't mention installing it. I assume this is related to `sed -i` not behaving as expected on a Mac? Maybe the command used needs to change depending on the OS?

Reply:
Sorry, this was a mistake. As you say, `sed -i` doesn't work on macOS, but I don't want to fix `sed` in this PR.